Add Anypoint monitoring instructions to 1.4 (#144)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/manage-monitor-applications.adoc

== Anypoint Monitoring

xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule runtime engine (Mule) flows and components in your application network. For applications and API gateways deployed to Anypoint Runtime Fabric (Runtime Fabric), Anypoint Monitoring provides basic metrics to all customers. Titanium-plan customers have access to additional capabilities, including application log retrieval and search.

> 

Reviewer: Are there any more useful details to add to this introduction that address the audience as "you" and tell "you" exactly why "you" wants or needs the following information to accomplish a specific user task or goal or to solve a particular user problem?

> === Enable Anypoint Monitoring

Reviewer: This isn't really a task topic, and the title implies that it is.
I'd say that this is more Requirements for Monitoring

Reviewer: ...and now that I have read the topic, this really isn't about enabling, because that is the default.
It is about ensuring the prerequisite of an outbound connection is available. 
Consider tweaking the wording to clarify, because users really don't need to enable monitoring.

> By default, Runtime Fabric is configured to send application metrics to Anypoint Monitoring. Titanium-plan customers also have application logs automatically sent to Anypoint Monitoring by default. For metrics and logs to be sent to Anypoint Monitoring, outbound network connectivity must be available. 

Reviewer: ```suggestion
By default, Runtime Fabric is configured to send application metrics to Anypoint Monitoring for all customers, provided that outbound network connectivity is available. 
```

> If you need an outbound proxy, configure Runtime Fabric to use a SOCKS5 proxy by following the instructions in xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric].

Reviewer: Shouldn't this follow the verification information?

Reviewer: ```suggestion
If you need an outbound proxy, follow the instructions in xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric].
```

> To verify outbound network connectivity, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 5044 to:

Reviewer: ```suggestion
Verify outbound network connectivity by ensuring that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 5044 to:
```

. US control plane: `dias-ingestor-nginx.prod.cloudhub.io`
. EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`

=== Disable Anypoint Monitoring for an Application

> When you deploy an application to Runtime Fabric, Anypoint Monitoring is enabled by default. To disable Anypoint 

Reviewer: Do we really need to restate this?
What would be useful here would be some text that tells the audience under what conditions it wants or needs to disable the default: WHY, to what advantage or to solve what problem?

> Monitoring, pass the following custom property when deploying your application:

Reviewer: I'm curious about "pass the property": is that the same thing as "include"? "Include" might be a better term to use, because nonnative English readers might mistake "pass" for "forego" or "omit."
